android/app/build.gradle

```diff
@@ -160,6 +160,7 @@ dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
+    compile 'com.facebook.fresco:animated-gif:0.11.0' // For animated GIF support
```

Comment:
ok cool 👍! so we can add this at that time. For now we can remove this, using unwanted dependencies is not good (makes app heavy).